src/commands/scrapers/FCCCommand.js

```diff
@@ -72,8 +72,8 @@ function removeHTMLTags(textArray) {
     .replace(/<code>|<\/code>/g, '`')
     .replace(/<blockquote>|<\/blockquote>/g, '```')    
     .replace(/<b>|<\/b>|<strong>|<\/strong>/g, '**')
-    .replace(/<i>|<\/i>/g, '*')
-    .replace(/<br>/g, '\n')
+    .replace(/<i>|<\/i>|<em>|<\/em>|<dfn>|<\/dfn>/g, '*')
+    .replace(/<br>/g, '\n')    
```

Comment:
Could you remove this trailing whitespace?

Reply:
There's trailing white space all over the place. Is there something special about that particular trailing white space?

Reply:
Yes -- that one happened to catch my eye 😛.

At some point in the distant feature, we should clean the whitespace up. However, I guess we should do it in one go in another PR, when we add linting. LGTM